add skip_load_external

TZanke opened the pull request:
Fixes https://github.com/peterbe/premailer/issues/207

Please review my code again.

peterbe commented on premailer/premailer.py:
```diff
@@ -540,6 +544,8 @@ def _load_external_url(self, url):
     def _load_external(self, url):
         """loads an external stylesheet from a remote url or local path
         """
+        if self.skip_load_external:
```
Since it's not many places where `self._load_external()` is called, I think you can be confident it won't get here if `skip_load_external` *is* `False`.

TZanke commented:
I implemented your suggestions and renamed parameter to `allow_network` which defaults to True. But i would like to set it to False, because in corporate environments we prefer security related settings to be as secure as possible by default. At the moment, without this branch, i ended up in ExternalNotFoundError, because network access is limited. Without our firewalls, premailer would download unknown files to our systems without notice.

peterbe commented:
Great work! I like it. Just one last thing; can you add a unit test that uses `allow_network=False`?

Also, regarding the default, I doubt it's feasible to change the default to `False` because it's a breaking change and it's rather strange that a server isn't allowed to reach for things on the Internet. Although I work in a security paranoid enterprise myself (Mozilla), I've never heard of this concern. But I don't dispute it. 

Perhaps you can add a blurb to the README that says "Warning! By default, `premailer` will attempt to download any external stylesheets by URL over the Internet. If you want to prevent this you can use the `allow_network=False` option."

peterbe commented:
Ping for a unit test added to the PR.

peterbe commented on README.rst:
```diff
@@ -61,6 +61,12 @@ premailer does this. It parses an HTML page, looks up ``style`` blocks
+
+    Warning!
+    By default, premailer will attempt to download any external stylesheets by URL over the Internet.
+    If you want to prevent this you can use the allow_network=False option.
```
```suggestion
    If you want to prevent this you can use the `allow_network=False` option.
```

peterbe replied:
```suggestion
    If you want to prevent this you can use the ``allow_network=False`` option.
```
